### DISCUSSION TITLE
eslint gives me an error or a big json file

### DISCUSSION THREAD

**Discussion by LupuC** [eslint/eslint#17333] [Q&A] [answered]
How can I disable ESlint for a specific .json file in nextjs app?
I've tried to create a file .eslintignore, to add in .eslintrc.json, in tsconfig.json and none of them seems to work. The error is:
```bash
./folder/jsonfile.json:1:2754821
Type error: Property value can only be string literal, numeric literal, 'true', 'false', 'null', object literal or array literal.
```
The json is in format:
```json
{"idlist":{"ids":[
{"id":1234,"name":"name here"},
{"id":234,"name":"name here"},
{"id":552,"name":"name here"}
```

and I import it inside a page, like this:
```js
import idlist from "@/folder/jsonfile.json"
```

Note that the file is pretty big (like 8MB of data).

edit, what I have tried:

.eslintignore with:

```
folder/*.json
folder/**.json
folder/
folder/jsonfile.json
/folder/
/folder/jsonfile.json
folder/*
folder/**

```

adding ```/* eslint-ignore */``` at the top of the json file

adding to package.json eslint like here https://eslint.org/docs/latest/use/configure/ignore

**Comment by nzakas** [comment, marked as answer]
ESLint itself doesn't actually support linting JSON files, so this message is coming from a plugin, and I have no idea what NextJS is doing by default.

I'm not sure what NextJS is resolving `@` to, but you may want to try this in your `.eslintignore` file:

```
**/folder/*.json
```

If that still doesn't work, I'd suggest asking this question in a NextJS-related forum.
